shared/engine/index.js

```diff
@@ -21,11 +21,18 @@ class Engine {
     windowsHack()
     setupLocalLogs()
 
+    // Keep some meta data from session ID to response meta
+    // To help debug outstanding requests
+    this.responseMeta = {}
+
     if (printOutstandingRPCs) {
       setInterval(() => {
         const keys = Object.keys(this.sessionIDToResponse).filter(k => this.sessionIDToResponse[k])
         if (keys.length) {
-          console.log('Outstanding RPC sessionIDs: ', keys)
+          keys.forEach(k => {
+            const {method, param} = this.responseMeta[k]
+            logLocal('Outstanding RPC sessionIDs: %s.\nMethod: %s with param: %O', k, method, param)
```

Comment:
I think this is much more helpful than just a number and having to look around at logs to figure out what method it was